### DISCUSSION TITLE
diary/2023/12/17/live-like-a-game/

### DISCUSSION THREAD

**Discussion by giscus** [zzsza/blog-comments#150] [Comments]
# diary/2023/12/17/live-like-a-game/

키워드 : 인생의 태도, 게임처럼 살자, 인생을 게임처럼, 삶의 태도 인생을 바라보는 태도 인생을 살다보면 다양한 경험을 하기도 하고, 좋은 일도 있고 힘든 일도 존재합니다 최근 받은 질문 : 힘들 때 어떤 생각을 하시나요? 벅찬 세상을 어떻게 살아가야 할 지 모르겠어요 이 부분에 대한 답은 저도 정확힌 알 수 없지만,...

https://zzsza.github.io/diary/2023/12/17/live-like-a-game/

**Comment by witch-factory** [comment]
잘 읽었습니다! 제게 해당되는 내용들도 있어서, 당장 어떻게 해야 할지 확 떠오르지는 않지만 제 상황들에 대해 다시 생각해 보게 하는 지점들이 좀 있네요. 감사합니다.
다만 읽다가 약간 이상했던 부분이 있었는데, ‘이벤트’ 섹션에서 2-3년 전에 개발자 연봉이 높은 것도 일종의 퀘스트라고 볼 수 있고 부분의 퀘스트 -> 이벤트 로 수정되어야 할 것 같습니다.

**Reply by zzsza** [reply]
안녕하세요! 이벤트 섹션에서 제가 다른 단어를 쓴 것이 맞네요. 피드백 해주셔서 감사합니다! 2시간 안에 글을 쓰려고 타임어택하느라 정말 빠르게 글을 작성했는데, 그러다보니 이렇게 되었네요. 수정하는 김에 문장들 전체를 다 보고 수정했어요! 
좋은 피드백 주셔서 너무 감사합니다.
당장 어떻게 해야할 지는 게임에서 내가 어떻게 할 수 있을까? 생각해보고 -> 현실에선 어떤 관점으로 볼 수 있지? 생각해보면 조금 나아지기도 하더라구요! 하나씩 생각해보셔요

**Comment by Toddhee** [comment]
글 재미나게 잘 읽었습니다!
구글링을 하다보니 비슷한 컨셉의 글을 하나 찾았는데 흥미로워서 공유드려봅니다. '돈'에 대해서도 짤막하게 다루고 있는데 재밌네요.
https://oliveremberton.com/2014/life-is-a-game-this-is-your-strategy-guide/

**Reply by zzsza** [reply]
오 좋은 글 공유해주셔서 감사합니다! 저도 게임의 재화에 대해서도 생각했었는데, 글을 작성할 당시엔 작성하지 못했네요. 조만간 재화에 대해서도 생각을 작성해봐야겠어요!! 감사합니다 :)

**Comment by airborne3586** [comment]
좋은 글이네요 데이터 분석가로 취준하고 있는 제 관점에서 포스팅 해보면 재밌을거같아서 써보려구요^^ 
PM강의 잘듣고 있습니당~

**Reply by zzsza** [reply]
좋은 글이라고 해주셔서 감사합니다! 강의도 듣고 계시는군요..! 도움이 되길 바랄게요! :)

**Comment by archmond** [comment]
너무 좋은 글이네요. 인생을 바라보는 시선을 달리하는 것 만으로도 흥미롭습니다. 도전에 대한 불안이 생기면 이 글을 보면 해쳐나갈 힘이 생길 것 같습니다.
북마크하고 어떤 역경이 있을 때 참고하겠습니다. 고맙습니다.

**Reply by zzsza** [reply]
불안이 생긴다면 다시 이 글을 보고 "에잇 그래도 이 게임 내가 다 끝장내버리겠어!"라고 생각하면 더 좋더라구요.
우리 모두 파이팅이에요! :)

**Comment by insam16** [comment]
안녕하세요 선생님! 좋은 글 잘 봤습니다. 막막하다고 느끼던 와중에 글을 읽고, 내가 정체기에 있구나. 올리기 어려운 레벨이구나 싶어서 위로를 받았습니다. 앞으로도 화이팅합시다!

**Reply by zzsza** [reply]
이 글을 통해 위로를 받았다니 너무 영광이에요. 우리 정체기 구간을 한번 잘 지나가보아요!

**Comment by singo112ok** [comment]
재밌으며 좋은글이네요! 링크주소만 퍼갈께요!

**Reply by zzsza** [reply]
재밌고 좋다고 해주셔서 감사합니다! 마음껏 공유해주셔도 좋아요 :)

**Comment by Lee-Soyeon** [comment]
왕재밌는 생각입니다~! 인생을 재밌게 살고 싶었는데 딱 좋은 방법 같아요 :)

**Reply by zzsza** [reply]
재미있는 생각이라 해주셔서 감사합니다. 인생을 이왕이면 즐겁게 생각해보면 더 좋은 것 같더라구요!

**Comment by Yerin99** [comment]
좋은 인사이트가 담긴 글이네요 :)

**Comment by DSeung001** [comment]
재밌네요 늘 인생은 게임 같다고 생각했는데
이런 식으로 정리한 걸 보니 또 감회가 새롭네요
